src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx

```diff
@@ -0,0 +1,77 @@
+import {
```

Comment:
Make payment button is appearing only when GPay is selected. If it's not intended do check for the issue.

![image](https://user-images.githubusercontent.com/91944618/159241348-247f1b1c-a908-40d0-9536-e2067297dab8.png)

Reply:
Its working fine now.

Reply:
It's a feature. To proceed further for checking out, the user would need to select a delivery address and a payment method as also mentioned just below the final amount. 
Failure to select any of the options, the user will not be able to make payment.